pages/index.js

```diff
@@ -1,17 +1,24 @@
-import Head from "next/head";
-import Navbar from "../components/Navbar";
-import About from "../components/About";
-import Join from "../components/Join";
+//Components - these provide styling and layout for the page
+import Layout from "../components/Layout";
+import Hero from "../components/Hero";
+
+//Content - this is the detail on the page
+import content from "../content/homepage";
+
 export default function Home() {
+  //Set up our SEO for the page
+  const seo = {
```

Comment:
We'll probably want to add more SEO info here. We could also add this to the 'content' file and then just pass `seo={content.seo}` if we wanted to